category_theory/category.lean

```diff
@@ -0,0 +1,44 @@
+-- Copyright (c) 2017 Scott Morrison. All rights reserved.
```

Comment:
The header we use look like:

```lean
/-
Copyright (c) 2014 Microsoft Corporation. All rights reserved.
Released under Apache 2.0 license as described in the file LICENSE.
Authors: Leonardo de Moura
notation, basic datatypes and type classes
-/
```

so use `/--/` instead of `--`. The difference looks minor, but might be helpful when we use scripts to analyze the header.